CI/devopsIntTest.ps1

```diff
@@ -34,4 +34,14 @@ else {
     Write-Host "Done testing SDP yml..."
 }
 
+& "$repoRoot/ECMA2Yaml/ECMA2Yaml/bin/Release/ECMA2Yaml.exe" -s $repoRoot\test\xml -o $repoRoot\test\yml_UWP_SDP_After --UWP --SDP -f --repoRoot $repoRoot\\ --repoBranch master --repoUrl https://github.com/docascode/ECMA2Yaml
+& "$repoRoot/ECMA2Yaml/DiffFiles/bin/Release/DiffFiles.exe" -o $repoRoot\test\yml_UWP_SDP -n $repoRoot\test\yml_UWP_SDP_After -l $repoRoot\test --Path
```

Comment:
same as the other PR, if it's a temp folder, start the name with `_yml` so it can be ignored.